ci/playbooks/hive/claim.yml

```diff
@@ -0,0 +1,6 @@
+- name: Install deps and prepare for KUTTL run
+  hosts: ""
```

Comment:
What should be the hosts here?

Reply:
I think it should be `localhost` ? i.e. the host running ansible would request the cluster. Would that make sense?